crosslink/cmd/root.go

```diff
@@ -123,14 +134,16 @@ func Execute() {
 }
 
 func init() {
-
 	comCfg.rootCommand.PersistentFlags().StringVar(&comCfg.runConfig.RootPath, "root", "", `path to root directory of multi-module repository. If --root flag is not provided crosslink will attempt to find a
 	git repository in the current or a parent directory.`)
-	comCfg.rootCommand.PersistentFlags().StringSliceVar(&comCfg.excludeFlags, "exclude", []string{}, "list of comma separated go modules that crosslink will ignore in operations."+
-		"multiple calls of --exclude can be made")
 	comCfg.rootCommand.PersistentFlags().BoolVarP(&comCfg.runConfig.Verbose, "verbose", "v", false, "verbose output")
+	comCfg.rootCommand.Flags().StringSliceVar(&comCfg.excludeFlags, "exclude", []string{}, "list of comma separated go modules that crosslink will ignore in operations."+
+		"multiple calls of --exclude can be made")
 	comCfg.rootCommand.Flags().BoolVar(&comCfg.runConfig.Overwrite, "overwrite", false, "overwrite flag allows crosslink to make destructive (replacing or updating) actions to existing go.mod files")
 	comCfg.rootCommand.Flags().BoolVarP(&comCfg.runConfig.Prune, "prune", "p", false, "enables pruning operations on all go.mod files inside root repository")
+	comCfg.pruneCommand.Flags().StringSliceVar(&comCfg.excludeFlags, "exclude", []string{}, "list of comma separated go modules that crosslink will ignore in operations."+
```

Comment:
Sorry I thought I left this comment in my initial review. 

Is this additional flag used in any of the pruning logic? I do not see any updates to `prune.go`.

Reply:
Yes. It is used here: https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/6d04deb6001c74346ccd15543da4e7261b922ebc/crosslink/internal/prune.go#L60 + https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/6d04deb6001c74346ccd15543da4e7261b922ebc/crosslink/cmd/root.go#L44